portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/moresites.vm

```diff
@@ -71,11 +69,11 @@
                     #macro( displaySite $site )
                         <li role="presentation" class="fav-sites-entry #if (${site.isCurrentSite})is-selected #end #if (${site.isMyWorkspace})my-workspace #end">
                             #if (!${site.isMyWorkspace})
-                            	<a class="site-favorite-btn" data-site-id="${site.siteId}" href="javascript:void(0);"></a>
+                                <a class="site-favorite-btn" data-site-id="${site.siteId}" href="#"></a>
```

Comment:
can we make this a button?

Reply:
Sure, I can make a transparent style for it.